cfgov/v1/models/molecules.py

```diff
@@ -78,7 +78,7 @@ class FormFieldWithButton(blocks.StructBlock):
     btn_text = blocks.CharBlock(max_length=100, required=False)
 
     required = blocks.BooleanBlock(required=False)
-    id = blocks.CharBlock(max_length=100, required=False)
+    id = blocks.CharBlock(max_length=100, required=False, help_text="Type of form i.e emailForm, submission-form")
```

Comment:
so it will show up underneath the field. i'm all up for suggestions on formulating the text better
<img width="1140" alt="screen shot 2015-12-28 at 12 12 49 pm" src="https://cloud.githubusercontent.com/assets/254877/12022297/6c9c4db8-ad5c-11e5-8795-d001a9e09da2.png">